src/common/users.ts

```diff
@@ -441,17 +448,39 @@ export const checkUserStreak = (streak: GQLUserStreakTz): boolean => {
   today.setHours(0, 0, 0, 0);
 
   const day = today.getDay();
-  const difference = differenceInDays(today, lastViewAt);
+  const difference = differenceInDays(today, lastStreakUpdate);
 
   return shouldResetStreak(day, difference, streak.weekStart);
 };
 
+export const getUserLastStreak = async (
```

Comment:
This is really getLastStreakRecoverDate no?

Reply:
Yep, that's more accurate.